cumulusci/robotframework/tests/salesforce/browsers.robot

```diff
@@ -0,0 +1,77 @@
+*** Settings ***
+
+Resource        cumulusci/robotframework/Salesforce.robot
+Suite Teardown  Close all browsers
+
+
+*** Variables ***
+${DEFAULT WINDOW SIZE}  1280x1024
+
+*** Keywords ***
+Assert active browser count
+    [Documentation]   Assert that an expected number of browsers are active
+    [Arguments]       ${expected count}
+
+    @{browsers}=      get active browser ids
+    ${actual count}=  get length  ${browsers}
+    Length should be  ${browsers}  ${expected count}
+    ...  Expected to find ${expected count} open browsers, found ${actual count}
+
+Assert window size
+    [Documentation]
+    ...  Verify the actual window size is the expected size
+    ...  give or take a pixel or five.
+    [Arguments]  ${expected width}  ${expected height}
+
+    # On circleci, the browser size is sometimes off by a pixel. How rude!
+    # Since we don't so much care about the precise size as we do that
+    # we are able to change the size, we'll allow a tiny bit of wiggle room.
+    ${actual width}  ${actual height}=  Get window size
+    ${xdelta}=  evaluate  abs(int($actual_width)-int($expected_width))
+    ${ydelta}=  evaluate  abs(int($actual_height)-int($expected_height))
+    Run keyword if  $xdelta > 5 or $ydelta > 5
+    ...  Fail  Window size of ${actual width}x${actual height} is not close enough to expected ${expected width}x${expected height}
+
+*** Test Cases ***
+Open Test Browser Twice
+    [Documentation]  Verify that we can open two browsers in a single test
+    [Tags]  issue:1068
+    [Teardown]  Close all browsers
+
+    Assert active browser count  0
+    Open test browser
+    Open test browser
+    Assert active browser count  2
+
+Browser aliases
+    [Documentation]  Verify that aliases are properly handled in Open Test Browser
+    [Tags]  issue:1068
+    [Setup]  Run keywords
+    ...  Open test browser  alias=browser1
+    ...  AND  Go to  https://metadeploy.herokuapp.com/products
+    ...  AND  Open test browser  alias=browser2
+    ...  AND  Go to  https://mrbelvedereci.herokuapp.com/
+    [Teardown]  Close all browsers
```

Comment:
At first I thought it was neat that this also implicitly tests that MetaDeploy and MetaCI are up, but since those tools are based on CumulusCI, I want to avoid the situation where one of them goes down due to a bug in CumulusCI and we can't merge the fix because this test is failing. Can we switch to sites that don't depend on CumulusCI?

Reply:
yeah, I've been second guessing myself on that. Have any recommendations? I want something where I can hard-code the URL for simplicity. I was thinking google.com and bing.com, but that's kinda rude to use someone else's servers in a test. I also though of maybe a couple of static pages on salesforce.com.